.goreleaser.yaml

```diff
@@ -0,0 +1,494 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+project_name: bao
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+    - make bootstrap
+
+builds:
+  - id: builds-linux
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{ .CommitDate }}
```

Comment:
This generate stripped builds which is incompatible with upstream's binaries which ship debug information. As I commented in https://github.com/hashicorp/vault/issues/21069, several people rely on the debug symbols; I can't remember the name of the product but there's a (popular among a subset of customers) product that instruments binaries in production and aggregates performance and other information.

Reply:
Stripping those out then!